### DISCUSSION TITLE
RDP Connection Error

### DISCUSSION THREAD

**Discussion by luqijun** [rustdesk/rustdesk#9561] [Q&A]
I have just used rustdesk for a short time and would like to ask a question.
When I use the RDP function, it seems to be connected to another computer, but it exits with the following error:

_Remote Desktop Connection
Your Remote Desktop Services session has ended.
The connection to the remote computer was lost, possibly due to network connectivity
problems, Try connecting to the remote computer again, lf the problem continues, contact
your network administrator or technical support._

![482b183b624812bf9f37824ba8c9a10f](https://github.com/user-attachments/assets/1c323c89-11ac-4a06-b36d-54c1bede0b77)
How to solve this problem？

**Comment by megyfexid** [comment]
Rustdesk Client RDP Compatibility Issue and Fix
Problem
When the Rustdesk client (Rustdesk.exe) is installed as a Windows service, Remote Desktop Protocol (RDP) functionality becomes non-operational.
Solution
To maintain RDP functionality with the Rustdesk client:

Do not install the Rustdesk client as a Windows service
Instead, run Rustdesk.exe directly on the remote machine

Important Clarification

This issue is specifically related to the Rustdesk client application
The Rustdesk server components (hbbs.exe and hbbr.exe) are not affected by this problem
Only the client-side installation method impacts RDP functionality

Detailed Steps

Avoid using the Install button in the Rustdesk client
Do not use the Rustdesk client msi installer
Simply download the Rustdesk client executable
Run Rustdesk.exe manually when remote access is needed
Ensure the Rustdesk client is running, but not installed as a system service

**Reply by megyfexid** [reply]
Sorry scratch this. It worked for a few hours but now it has stopped again.

**Comment by poffio** [comment]
Hi, I've the same issue on Rustdesk Pro 1.4.8 and client 1.3.7.
Do you have any plan to fix this issue in future versions ? Thank You, Fabio.

**Comment by lesourcil** [comment]
Same issue here.

**Comment by Cerberusfall** [comment]
Getting the same issue, randomly it just started happening.

**Comment by cidzoo** [comment]
Same here. 

Does an issue already exists for this problem?

Has anyone found a workaround that doesn't implies a no-install run? 

Running rustdesk without installation doesn't allow switching between multiple windows accounts on the remote machine. Because then RustDesk only run in the context of the currently logged-in user and won't survive a logout. I've also tried to run it using Task Manager when computer starts, both as an admin account or SYSTEM account, but it's doesn't work either.

**Reply by cidzoo** [reply]
I've created the issue #11483 to report this problem.

**Reply by rustdesk** [reply]
Please do not create duplicate things. RustDesk's rdp is just a plain port forwarding, we do not know how to fix it. It might be related to https://github.com/rustdesk/rustdesk/issues/487.

**Reply by cidzoo** [reply]
@rustdesk As you said it might be related to #487. But it's far from being a duplicate. It's why I took time and effort to fill-in a detailed issue to report this problem.

Nonetheless you've simply deleted (not even closed) my issue report I've just made. Why?

**Reply by rustdesk** [reply]
According to our policy https://github.com/rustdesk/rustdesk/discussions/763#discussioncomment-11527477

**Reply by cidzoo** [reply]
Too bad because my issue was far more detailed and explicit than this discussion. 

I don't really get the policy of "No issue duplicate to the discussion". Apparently [I'm not the only one](https://github.com/rustdesk/rustdesk/discussions/10127). 

A _discussion_ is designed, as the name suggests, to first discuss something. In our case, a potential problem. Then, if the discussion seems to confirm the problem with multiple reports, it can be **transitioned** to an _issue_. The goal being to summarize the discussions in a clear and concise way, give the precise steps to reproduce it, etc. Finally the discussion could be closed and reference the newly created issue.

What's the point of throwing away a precise and concise issue while keeping open the initial discussion? Moreover other people who come across this discussion can't even view the issue because you simply deleted it. Maybe someone else, with the needed skills, is willing to solve this ?

If it still doesn't convince you and you don't want to reconsider it, could you at least undelete it and close it?

**Comment by poffio** [comment]
<img width="255" height="257" alt="image" src="https://github.com/user-attachments/assets/80b9739b-14cc-4f24-be2d-7077f3aecc65" />

Same problem here. RDP connection via tcp tunnel sometimes works sometimes it doesn't. With same configuration.

Thank You,
Fabio.

**Reply by poffio** [reply]
@rustdesk thanks for your support. Could the RDP problem be related to what is indicated here [#12936](https://github.com/rustdesk/rustdesk/discussions/12936) ?

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/pull/12881 ready in nightly build
